Framework/Geometry/src/Instrument/InstrumentDefinitionParser.cpp

```diff
@@ -3008,6 +3008,12 @@ InstrumentDefinitionParser::convertLocationsElement(
         Strings::strip(pElem->getAttribute("name-count-start")));
   }
 
+  int nameCountIncrement(1);
```

Comment:
Am I right in thinking that you support negative increments too? if not maybe make this unsigned?